src/parser/name_resolution.c

```diff
@@ -717,6 +717,11 @@ pt_bind_name_or_path_in_scope (PARSER_CONTEXT * parser, PT_BIND_NAMES_ARG * bind
 	    }
 	  level++;
 	}
+      if (node && er_errid () == ER_OBJ_INVALID_ATTRIBUTE)
+	{
+	  /* An error is meaningful only when it cannot be found in all scopes. */
+	  er_clear ();
+	}
```

Comment:
After calling pt_expand_external_path() on line 710, it seems more readable to check as follows.

```
      if (er_errid () == ER_OBJ_INVALID_ATTRIBUTE)
	{
	  /* An error is meaningful only when it cannot be found in all scopes. */
	  er_clear ();
	}
```

Reply:
I think checking outside the 'for loop' looks good too. I'll keep this source code.